website/package.json

```diff
@@ -19,6 +19,7 @@
     "@stoplight/elements": "^7.7.17",
     "@svgr/webpack": "^6.0.0",
     "axios": "^0.27.2",
+    "build": "^0.1.4",
```

Comment:
Was it intended to add this `build` package? It looks like the snyk errors are coming from this package. If this is unneeded, removing this line from the `package.json` should resolve the snyk issues.
![image](https://github.com/dbt-labs/docs.getdbt.com/assets/13647110/7d980d4a-aac3-4c69-aa50-cdc3fa0a92fc)